tf2_tools/setup.py

```diff
@@ -0,0 +1,34 @@
+from setuptools import setup
+
+package_name = 'tf2_tools'
+
+setup(
+    name=package_name,
+    version='0.15.0',
+    packages=[package_name],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    author='Wim Meeussen',
+    maintainer='Chris Lalancette',
```

Comment:
Add me here as a maintainer too ?

Reply:
As far as I understand, there is no support for multiple maintainers in the setup.py format: https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py .

I guess I could add in a comma-separated string of maintainers/email addresses, but I don't think it is totally necessary.  Both of our names are in the package.xml, which is good enough for me.